### ISSUE TITLE
[Bug]: ABS does not add ebooks to its libraries in folders with metadata.opf files created by Calibre

### ISSUE THREAD

**Issue by iconoclasthero** [advplyr/audiobookshelf#2111]
### Describe the issue

Ubuntu 22.04, ABS 4.2.2, calibre 6.12
ABS does not add ebooks to its libraries in folders with metadata.opf files created by Calibre.  E.g., in my ebook libarary which is in the default calibre format: /library/ebooks/author/book/ebook.@(mobi|pdf|awz3|epub).  In that terminal directory, calibre adds a metadata.opf file.  I pointed ABS at that library and I only the last few books that I added (i.e., that hadn't been input into Calibre yet) showed up in ABS.  I deleted the library (after adding the few stragglers above to calibre) and recreated the ebook library.  0% of the books were added.  I went to one book and `$ mv metadata.opf ` and the book appeared.  This was observed with a second book.  

I don't know why abs is inhibited from adding books with the metadata.opf and I get that having two different applications managing the same library can cause issues...but I still want the books added to ABS.

### Steps to reproduce the issue

1. Install calibre and have it manage your ebooks
2. point abs at the ebook folders managed by calibre that have calibre's metadata.opf files in them
3. see that abs doesn't add books
4. remove metadata.opf file from ebook dir
5. see abs add book

### Audiobookshelf version

4.2.2

### How are you running audiobookshelf?

Debian/PPA

**Comment by advplyr** [comment]
I just tested this with a calibre folder and every book was detected. The metadata.opf should not have anything to do with the scanner issue you are having. Abs will use a lot of the data from the opf file.

It's possible that the reason removing the metadata.opf added the book is because the scanner detected a change in that folder and rescanned the folder. If that is the case then removing or adding any file would have caused the re-scan.

You will need to provide more information or narrow down the steps of reproducing this

**Comment by iconoclasthero** [comment]
Like I said, I tried to touch each and every single file/folder in that directory and it did not work.
I did a $ find. "metadata.opf" -exec mv {} {}\~ \;
and now it populates all the books.  I'll go back and use find to undo that later so that calibre has it's metadata.opf files again.

**Comment by advplyr** [comment]
Yes, but I also tested adding a new book through calibre and Abs automatically scanned it in correctly

**Comment by iconoclasthero** [comment]
Well, it's done scanning and now I have 565 books.  All I did was the above find ... -exec ... command and the rescanned it.

I am sure that it isn't a matter of how recently the files/folders were modified since I did something like `$ find * -iname "*" -exec touch {} \; `.  Besides, I told it to scan the folder and it only found things without metadata.opf files with the ebooks.  The age of the folders don't matter for any other books (which are all audiobooks without the metadata.opf files).
